manager/db-migrations/src/main/resources/changesets/15-add-expanded-transfer-status.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="14" author="raaid">
```

Comment:
```suggestion
    <changeSet id="15" author="raaid">
```

Reply:
Strange the tests don't flag this though...

Reply:
😱 good catch